packages/MarkdownEditor-Tests.package/MarkdownBlockParserTest.class/instance/headingFontSized..st

```diff
@@ -0,0 +1,7 @@
+accessing
+headingFontSized: pointSize
```

Comment:
Couldn't you create a `TextFontReference` here? All calls will wrap the font into it, which adds some bloat to the code imo